bin/pdd_renewal_report.pl

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+use utf8;
+
+BEGIN {
+  die "SDRROOT environment variable not set" unless defined $ENV{'SDRROOT'};
+  use lib $ENV{'SDRROOT'} . '/crms/cgi';
+}
+
+use Getopt::Long qw(:config no_ignore_case bundling);
+
+use CRMS;
+
+use constant {
+  EXPECTED_RENEWAL_SUNSET => 68 # years
+};
+
+binmode(STDOUT, ':encoding(UTF-8)');
+my $usage = <<END;
+USAGE: $0 [-hv] [-y YEAR]
+
+Typically run a week or so into January. Reports on CRMS Core determinations with a
+now-expired renewal date for the current year that are still closed.
+
+This is a postmortem report intended to be run in January after new rights
+from the PDD rollover have taken effect. There will generally be no reason to use
+the -y flag; it is included here for consistency and in the hope it may be useful
+for future development purposes.
+
+The renDate fields that qualify are based on a current year minus 68 years scheme
+(see the magic constant EXPECTED_RENEWAL_EXPIRATION_YEARS). I will not attempt to derive
+that here except to note it relates to renewal terms in some not-particularly-obvious way;
+it is not an off-by-two derivation from 70 years but something more subtle. See Kristina
+for more information if this needs to be revisited.
+
+-h             Print this help message.
+-v             Emit verbose debugging information. May be repeated.
+-y YEAR        Use some other value for YEAR other than the current year.
+END
+
+my $help;
+my $verbose;
+my $year;
+
+Getopt::Long::Configure('bundling');
+die 'Terminating' unless GetOptions(
+           'h|?'  => \$help,
+           'v+'   => \$verbose,
+           'y:s'  => \$year);
+
+if ($help) {
+  print $usage. "\n";
+  exit 0;
+}
+
+my $crms = CRMS->new(
```

Comment:
I'd move line 63 up above this block starting at line 58,
so that you cannot accidentally instantiate a `$crms` with an undefined `$verbose` (which I think it looks like the code currently allows.)

Maybe that's a fine thing to do, but perl being perl I like to be sure that vars are defined.

Reply:
It "doesn't matter" (I'm trying to get rid of that `verbose` parameter to the CRMS initializer since it really doesn't do anything) BUT you're right, for consistency it really should track the script's verbosity setting and I'm definitely fixing that.